### DISCUSSION TITLE
Hi bro

### DISCUSSION THREAD

**Discussion by iphone12616868** [doronz88/pymobiledevice3#397] [General]
In repair industry we need diagnost when we change display to bind new face id & and when the wifi chip is damage we need to bind new chip on it until iphone 11 there is a chines machin do that but the 12 generation and above there is nothing to do that .do you think it is possible to think about it and make it

**Comment by iphone12616868** [comment]
Thanks for your hard work

**Comment by doronz88** [comment]
Can you describe what you expect pymobiledevice3 to do? I couldn't figure it out from your description
